app/scripts/lib/app-start.js

```diff
@@ -214,7 +216,6 @@ define(function (require, exports, module) {
         const iframeChannel = new IframeChannel();
 
         iframeChannel.initialize({
-          metrics: this._metrics,
```

Comment:
Yep. I only noticed because at first I thought I'd created a circular set of references in the initialisation code. But then when I traced it through I found `metrics` wasn't used by the channels so problem solved.